lib/opencomponents.rb

```diff
@@ -1,3 +1,5 @@
+require 'rest-client'
```

Comment:
does this have the specific version of `rest-client` in the gemfile this relies on?